### ISSUE TITLE
Dashboard doesn't get updated with area name change. 

### ISSUE THREAD

**Issue by kbtbc** [UnownHash/Dragonite-Public#28]
To recreate:  Edit an existing area and change it's name, save it.   It will update correctly in the area list and database but the Dashboard Areas section will show the old name with 'Name not found, please open PR' ...

**Comment by jfberry** [comment]
Yes, also logs don't get updated.  It's a known issue that is unlikely to be fixed any time soon
